nix/default.nix

```diff
@@ -3,21 +3,16 @@
 import sources.nixpkgs {
   overlays = [
     (_: pkgs: {
-      go = pkgs.go_1_18;
+      flake-compat = import sources.flake-compat;
+      go = pkgs.go_1_21;
       go-ethereum = pkgs.callPackage ./go-ethereum.nix {
         inherit (pkgs.darwin) libobjc;
         inherit (pkgs.darwin.apple_sdk.frameworks) IOKit;
-        buildGoModule = pkgs.buildGo118Module;
+        buildGoModule = pkgs.buildGo120Module;
```

Comment:
If go target version is 1.21, shouldn't it be pkgs.buildGo121Module?

Reply:
changed to 1.21